### PR TITLE
Add otx v1.7 b566

### PR DESCRIPTION
**Pull request by mandrean**
#### Adding a new cask
- [X] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Commit message includes cask’s name.
- [X] `brew cask audit --download otx` is error-free.
- [X] `brew cask style --fix otx` left no offenses.
- [X] `brew cask install otx` worked successfully.
- [X] `brew cask uninstall otx` worked successfully.

### CODE REVIEW

**Comment by mandrean** [comment]
Question: Do we have to add the following caveats:

> If you would like to install the #{token} CLI tool as well, please run the following command:
> 
> ln -s '#{staged_path}/#{token}' '/usr/local/bin/#{token}

Or can we just add that as a postflight script and a zapper?

**Comment by adidalal** [comment]
Add it in as a binary - I think the sublime cask should have an example. 

No need to add it as an uninstall item if you define it as an artifact (like sublime), we'll take care of it

File: Casks/otx.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask 'otx' do
+  version '1.7'
```
Comment (adidalal) [inline Casks/otx.rb]: You can make this 1.7,b66 and use version.before and after_comma methods in url

**Comment by mandrean** [comment]
Awesome @adidalal! Thanks. Pushed fixes. Tested and works nicely with install/uninstall.

**Pull request merged**